config/cesm/machines/config_machines.xml

```diff
@@ -862,7 +862,7 @@ This allows using a different mpirun command to launch unit tests
     <OS>LINUX</OS>
     <COMPILERS>gnu</COMPILERS>
     <MPILIBS>mpich</MPILIBS>
-    <CIME_OUTPUT_ROOT>$CASEROOT/</CIME_OUTPUT_ROOT>
+    <CIME_OUTPUT_ROOT>$ENV{HOME}/$CASE</CIME_OUTPUT_ROOT>
```

Comment:
How about we go with /scratch/${USER} instead, which is close to Cheyenne?  I'm not fond of forcing it into the home directory, since if someone creates a case in /home/user/foobar/test1, this creates a confusing directory in /home/user/test1 with the bld/run directories, right?  Secondly, this should allow us to also map in /scratch/user directories for people who want to mount some other storage for run stuff (faster storage, perhaps, like an SSD).

This may require simple changes to check / make that directory if it doesn't exist, but I think that's entirely in the container process.

Reply:
So then you would create a /scratch file system in the container?  I'm fine with that.

Reply:
Either /scratch or /home/scratch seems fine.  Maybe one is better than the other, but I need to think about that more.  For now, this all looks good, so I'll approve and we'll return to this if we need to change it.  (You had the ${HOME}/scratch in a later commit I hadn't seen when going through them chronologically.)